docs/topics/Voice_Connections.md

```diff
@@ -208,9 +208,9 @@ Voice data sent to discord should be encoded with [Opus](https://www.opus-codec.
 
 ## Speaking
 
-To notify clients that you are speaking or have stopped speaking, send an [Opcode 5 Speaking](#DOCS_TOPICS_OPCODES_AND_STATUS_CODES/voice) payload:
+Speaking updates are used to update the **speaking modes** used by the client. This speaking mode is set using a [Opcode 5 Speaking](#DOCS_TOPICS_OPCODES_AND_STATUS_CODES/voice) payload. The client must send this at least once before sending audio to update the SSRC and set the initial speaking mode.
 
-The following flags can be used as a bitwise mask. For example `5` would be priority and voice.
+The following flags can be used as a bitwise mask. For example `5` would be priority and voice. The speaking mode should not be 0 to allow sending audio.
```

Comment:
Do not understand why this sentence is needed.

Reply:
Because you should never set it to 0, it only causes issues.